app/controllers/CreateRecordSuccessController.scala

```diff
@@ -33,7 +33,8 @@ class CreateRecordSuccessController @Inject() (
 ) extends FrontendBaseController
     with I18nSupport {
 
-  def onPageLoad: Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    Ok(view())
+  def onPageLoad(recordId: String): Action[AnyContent] = (identify andThen getData andThen requireData) {
+    implicit request =>
+      Ok(view())
```

Comment:
We can pass the recordId down to the view for link "Categorise now". This is good for now

Reply:
done